libs/platform/experience/src/mixins/layout.mixin.ts

```diff
@@ -196,26 +195,31 @@ export const LayoutMixin = <T extends Type<LitElement & LayoutAttributes>>(
           options: this.$options() as CompositionProperties,
         },
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        attrs: this.attributeFilter()!,
+        attrs: this[LayoutMixinInternals].$attributeFilter(),
         screen: this.$screen(),
       });
     }
 
-    protected template$ = new Subject<{
-      template: TemplateResult;
-      experience?: LayoutMixinRender['experience'];
-    }>();
-    protected $template = computed(() => this.template$, {
-      equal: (a, b) => JSON.stringify(a) === JSON.stringify(b),
-    });
+    protected $template = signal(
+      undefined as
+        | {
+            template: TemplateResult;
+            experience?: LayoutMixinRender['experience'];
+          }
+        | undefined,
+      {
+        equal: (a, b) => JSON.stringify(a) === JSON.stringify(b),
+      }
+    );
```

Comment:
Took me a while to understand this one. We could move the typing into the generic.

Reply:
I have no idea why, but it didn't work :( and I was trying different approaches.

Reply:
I had no issues with it, doing:

```ts
protected $template = signal<
      | {
          template: TemplateResult;
          experience?: LayoutMixinRender['experience'];
        }
      | undefined
    >(undefined, {
      equal: (a, b) => JSON.stringify(a) === JSON.stringify(b),
    });
```